include/wctype.h

```diff
@@ -17,6 +17,9 @@ typedef _PDCLIB_wint_t wint_t;
 
 // wctrans_t
 // wctype_t
```

Comment:
These comments are probably useless now (I assume they only hinted that the types should be defined here - which is being done now).

I'd be willing to merge once that is removed.

Reply:
Done.